apisix/cli/ngx_tpl.lua

```diff
@@ -233,6 +233,9 @@ http {
     {% end %}
     {% end %}
 
+    # error_page
+    error_page 500 502 503 504 @apisix_error_handler;
```

Comment:
what about the 4xx response codes?

Reply:
Now the 4xx situation like:

404: {"error_msg":"404 Route Not Found"}
401: {"code":4107,"message":"request unauthorized","data":null,"request_id":""}

We will return JSON, do this need to add an error page?
IMO, it doesn't need now.

Reply:
Yes, we do need it.

```
<html>
<head><title>403 Forbidden</title></head>
<body>
<center><h1>403 Forbidden</h1></center>
<hr><center>openresty</center>
</body>
</html>
```

Maybe we can add a link in the response first.